### DISCUSSION TITLE
convering pat file to bam file

### DISCUSSION THREAD

**Discussion by mackaay** [nloyfer/wgbs_tools#58] [Q&A] [answered]
Hi, wgbs_tools looks great. I am just wondering how could we convert pat files to bam files back?

**Comment by nloyfer** [comment, marked as answer]
Hi @mackaay, and thank you.
There is no way to convert a pat file back to bam, since most of the data in the bam file (the DNA sequence, bp quality, exact read start/length etc.) is discarded when a pat file is created.
This allows pat files to be 300x smaller than their corresponding bam files.
